src/main/java/com/marklogic/appdeployer/command/temporal/DeployTemporalAxesCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.marklogic.appdeployer.command.temporal;
+
+import com.marklogic.appdeployer.command.AbstractResourceCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.mgmt.ResourceManager;
+import com.marklogic.mgmt.temporal.TemporalAxesManager;
+
+import java.io.File;
+
+public class DeployTemporalAxesCommand extends AbstractResourceCommand {
+
+	private String databaseIdOrName;
+
+	public DeployTemporalAxesCommand() {
+		// TODO - verify that range element indexes exist before creation of temporal axes?
```

Comment:
I think this is fine - possible enhancement in the future if the Manage API doesn't give a very helpful error